nihms-native-assembler/src/main/java/org/dataconservancy/nihms/assembler/nihmsnative/ThreadedOutputStreamWriter.java

```diff
@@ -62,4 +63,8 @@ private void debugResources(List<PackageStream.Resource> resources) {
         resources.forEach(r -> LOG.debug(">>>> Assembling resource: {}", r));
     }
 
+    protected String nameResource(Resource resource) {
```

Comment:
Unit test coverage for this method would be nice, but I know there's no existing test at the moment for `ThreadedOutputStreamWriter`.  Maybe some javadoc here would be appropriate, and documenting that this method assumes that the supplied `Resource` is always `DepositFileType.supplemental`.

Reply:
What is the the policy for documenting overridden methods?  Should I begin with the content from the base class and augment it for this version?  I certainly should have included something to explain the intent and the parameters, so I'll begin with a copy unless I hear otherwise.

Reply:
I don't think there is a policy.  If I override a method, I often will start out with {@inheritDoc} then start a new paragraph or h4 heading and document the implementation.